com.sap.cloud.lm.sl.cf.web/src/test/java/com/sap/cloud/lm/sl/cf/web/resources/ConfigurationEntriesResourceTest.java

```diff
@@ -53,52 +54,65 @@ public ConfigurationEntriesResourceTest(TestCase<TestInput> test) {
             {
                 new PostRequestTest(new PostRequestTestInput("configuration-entry-04.xml"), "R:configuration-entries-resource-test-output-04.json"),
             },
             // (4)
             {
-                new GetRequestTest(new GetRequestTestInput(100, "configuration-entry-05.json"), "R:configuration-entries-resource-test-output-05.json"),
+                new PostRequestTest(new PostRequestTestInput("configuration-entry-05.xml"), "E:Target does not contain 'org' and 'space' parameters"),
             },
             // (5)
             {
-                new GetRequestTest(new GetRequestTestInput(100, "configuration-entry-06.json"), "R:configuration-entries-resource-test-output-06.json"),
+                new GetRequestTest(new GetRequestTestInput(100, "configuration-entry-05.json"), "R:configuration-entries-resource-test-output-05.json"),
             },
             // (6)
             {
+                new GetRequestTest(new GetRequestTestInput(100, "configuration-entry-06.json"), "R:configuration-entries-resource-test-output-06.json"),
+           },
+            // (7)
+            {
                 new DeleteRequestTest(new DeleteRequestTestInput(100), "R:configuration-entries-resource-test-output-07.json"),
             },
-            // (7)
+            // (8)
             {
                 new SearchRequestTest(new SearchRequestTestInput(Arrays.asList("foo:bar", "baz:qux"), "R:parsed-properties-01.json"), "R:configuration-entries-resource-test-output-08.json"),
             },
-            // (8)
+            // (9)
             {
                 new PutRequestTest(new PutRequestTestInput(100, "configuration-entries-resource-test-input-09.xml"), "R:configuration-entries-resource-test-output-09.json"),
             },
-            // (9)
+            // (10)
             {
                 new PutRequestTest(new PutRequestTestInput(100, "configuration-entries-resource-test-input-10.xml"), "R:configuration-entries-resource-test-output-10.json"),
             },
-            // (10)
+            // (11)
             {
                 new PutRequestTest(new PutRequestTestInput(100, "configuration-entries-resource-test-input-11.xml"), "E:A configuration entry's id cannot be updated"),
             },
-            // (11)
+            // (12)
             {
                 new SearchRequestTest(new SearchRequestTestInput(Arrays.asList("{\"foo\":\"bar\",\"baz\":\"qux\"}"), "R:parsed-properties-01.json"), "R:configuration-entries-resource-test-output-08.json"),
             },
-            // (12)
+            // (13)
             {
                 new SearchRequestTest(new SearchRequestTestInput(Arrays.asList("a"), "R:parsed-properties-01.json"), "E:Could not parse content query parameter as JSON or list"),
             },
+            // (14)
+            {
+                new PostRequestTest(new PostRequestTestInput("configuration-entry-06.xml"), "R:configuration-entries-resource-test-output-01.json"),
```

Comment:
Should there be a test case where the user specifies both "target-space" and "provider-target"? What would happen then? This seems to be a valid input scenario according to the xsd schema.

Reply:
It must take take parameters from provider-target section but i added test just in case.